src/components/ActionChip/ActionChip.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div>
+    <v-chip v-if="actionableItem.action === ActionTypes.ADDED"
+            x-small label color="primary" text-color="white">ADDED</v-chip>
+    <v-chip v-if="actionableItem.action === ActionTypes.EDITED"
+            x-small label color="primary" text-color="white">{{editedLabel(isCorrection)}}</v-chip>
+    <v-chip v-if="actionableItem.action === ActionTypes.REMOVED"
+            x-small label color="#E0E0E0" text-color="#212529">REMOVED</v-chip>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import { ActionTypes } from '@/enums'
+import { ActionableItemIF } from '@/interfaces'
```

Comment:
Should this be `@bcrs-shared-components/interfaces` ? (and also for enums)

So that, in the app, it gets the shared interfaces?

Alternately, the app could re-export the shared interfaces. I did this once to prevent having to change a ton of stuff in the app but I don't think it's a good pattern. I think it's better if THIS component imports the things it needs directly.

Thoughts?

Reply:
I'm not entirely sure on this, i believe we will only have issues if there is a conflict with an already existing file in the same project. 
I do agree though, will update these imports.

Reply:
These import changes actually break the build. 
I think these changes will create a little tech debt, 
We could leave this imports BUT this will require to clear the conflicting enum/interface out of repo where these are pulled in potentially. 
I'm not sure the best answer. I'd like to see the conflict here though so perhaps we can better solve this.